### ISSUE TITLE
HTML mathjax rendering for VSCode

### ISSUE THREAD

**Issue by korsbo** [korsbo/Latexify.jl#136]
We could use HTML and mathjax/katex to get latexstrings rendered in the plot pane of VSCode. 

A crude prototype:
```
using Latexify

function html_render(lstr)
  str = replace(lstr, "\$"=> "\$\$")
  hstr = """
  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
  $(str)
  """
  display(MIME("juliavscode/html"), hstr)
end

lhs = [:a, :b]
rhs = [7, 8]

html_render(latexify(:(hello/bye)))
html_render(latexify(lhs, rhs))
```

This seems to work nicely but it needs improvement and actual integration.

**Comment by korsbo** [comment]
Or maybe using Katex instead since its faster.
```
function html_katex(lstr)
  str = replace(lstr, "\$"=> "\$\$")
  hstr = """
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.css" integrity="sha384-AfEj0r4/OFrOo5t7NnNe46zW/tFgW6x/bCJG8FqQCEo3+Aro6EYUG4+cU+KJWu/X" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/katex.min.js" integrity="sha384-g7c+Jr9ZivxKLnZTDUhnkOnsh30B4H0rpLUpJ4jAIKs4fnJI+sEnkvrMWph2EDg4" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.12.0/dist/contrib/auto-render.min.js" integrity="sha384-mll67QQFJfxn0IYznZYonOWZ644AWYC+Pt2cHqMaRhXVrursRwvLnLaebdGIlYNa" crossorigin="anonymous"
    onload="renderMathInElement(document.body);"></script>
  $(str)
  """
  display(MIME("juliavscode/html"), hstr)
end

```

It does not, however, support `align` but only `aligned` so something would have to be done about that.

**Comment by korsbo** [comment]
Closed by #137
